src/package/nuspec/netcoreapp/Microsoft.NET.Test.Sdk.targets

```diff
@@ -0,0 +1,49 @@
+<!--
+ ***********************************************************************************************
+ Microsoft.NET.Test.Sdk.targets
+
+ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+           created a backup copy.  Incorrect changes to this file will make it
+           impossible to load or build your projects from the command-line or the IDE.
+
+ Copyright (c) .NET Foundation. All rights reserved. 
+ ***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
```

Comment:
This is already defined in props file, is there a reason we need to redefine it here, if not, can you please remove it.